programs/fileio.c

```diff
@@ -2220,8 +2210,18 @@ FIO_determineDstName(const char* srcFileName)
 
     /* return dst name == src name truncated from suffix */
     assert(dstFileNameBuffer != NULL);
-    memcpy(dstFileNameBuffer, srcFileName, sfnSize - suffixSize);
-    dstFileNameBuffer[sfnSize-suffixSize] = '\0';
+    size_t dstFileNameEndPos = sfnSize - suffixSize;
```

Comment:
You're failing the contbuild because you declare this variable after other statements, which is not allowed in C90.